components/sync_manager/README.md

```diff
@@ -0,0 +1,13 @@
+# Sync Manager
+
+It's a bit unfortunate this component can't just be part of `sync15`.
+Conceptually and functionally, it shares most in common with with the `sync15`
+crate, and in some cases stretches (or arguably breaks) the abstraction barrier
+that `sync15` puts up.
+
+Unfortunately, places/logins/etc depend on sync15 directly, and so to prevent a
```

Comment:
It seems like some of the issue here WRT stretching/breaking the abstractions is keeping sync_multiple in sync15, but logins (and even places) probably doesn't actually need that?